pkg/minikube/machine/drivers/none/none.go

```diff
@@ -96,10 +91,19 @@ func (d *Driver) GetURL() (string, error) {
 }
 
 func (d *Driver) GetState() (state.State, error) {
-	command := `sudo systemctl is-active localkube 2>&1 1>/dev/null && echo "Running" || echo "Stopped"`
+	var localkubeStatusCommand = fmt.Sprintf("if [[ `systemctl` =~ -\\.mount ]] &>/dev/null; "+`then
```

Comment:
This might be out of scope for this fix, but why do we have this variable in two places?